pnpm-lock.yaml

```diff
@@ -26011,7 +26293,3 @@ packages:
     transitivePeerDependencies:
       - '@babel/core'
       - supports-color
-
```

Comment:
I don't know why this has gone from the lock file but I assume that's not a good thing,